.gitignore

```diff
@@ -72,6 +72,7 @@ tools/FAKE.Core
 tools/SourceLink.Fake
 tools/xunit.runners
 *.ncrunch*
+*.GhostDoc.xml
```

Comment:
It's a really nice tool when having to do a lot of XML docs :smile: The file is for custom settings and ignore words etc. for spell checking. GhostDoc generates it automatically for each solution I open, so I figured I might as well ignore it so I don't accidentally commit it. If you want it in the repo, I'll drop the commit :smile:

Here's a default file:

```xml
<GhostDoc>
  <SpellChecker>
    <IncludeExtensions>
    </IncludeExtensions>
    <IgnoreExtensions>
    </IgnoreExtensions>
    <IgnoreFiles>
    </IgnoreFiles>
  </SpellChecker>
<HelpConfigurations selected="HelpFile">
  <HelpConfiguration name="HelpFile">
    <OutputPath>.\Help</OutputPath>
    <HelpFileName>Bankeren</HelpFileName>
    <ImageFolderPath />
    <HtmlFormats>
      <HtmlHelp>true</HtmlHelp>
      <MSHelpViewer>false</MSHelpViewer>
      <MSHelp2>false</MSHelp2>
      <Website>false</Website>
    </HtmlFormats>
    <IncludeScopes>
      <Public>true</Public>
      <Internal>false</Internal>
      <Protected>false</Protected>
      <Private>false</Private>
      <Inherited>true</Inherited>
      <InheritedFromReferences>true</InheritedFromReferences>
      <EnableTags>false</EnableTags>
      <TagList />
    </IncludeScopes>
    <ResolveCrefLinks>true</ResolveCrefLinks>
    <HeaderText />
    <FooterText />
    <SelectedProjects />
  </HelpConfiguration>
</HelpConfigurations>
</GhostDoc>
```